spec/controllers/tasks_controller_spec.rb

```diff
@@ -344,4 +344,95 @@
       end
     end
   end
+
+  # rubocop:disable RSpec/MultipleExpectations
+  describe '#import_uuid_check' do
+    let!(:task) { create(:task, valid_attributes) }
+    let(:account_link) { create(:account_link, user: user) }
+    let(:uuid) { task.reload.uuid }
+    let(:post_request) { post :import_uuid_check, params: {uuid: uuid} }
```

Comment:
This is the test subject, right? We might also consider [naming it accordingly](https://relishapp.com/rspec/rspec-core/docs/subject/explicit-subject)